src/building/construction.c

```diff
@@ -572,7 +574,8 @@ void building_construction_update(int x, int y, int grid_offset)
         building_rotation_get_offset_with_rotation(10, building_rotation_get_rotation(), &x_offset_2, &y_offset_2);
         if (map_building_tiles_are_clear(x, y, 5, TERRAIN_ALL) &&
             map_building_tiles_are_clear(x + x_offset_1, y + y_offset_1, 5, TERRAIN_ALL) &&
-            map_building_tiles_are_clear(x + x_offset_2, y + y_offset_2, 5, TERRAIN_ALL)) {
+            map_building_tiles_are_clear(x + x_offset_2, y + y_offset_2, 5, TERRAIN_ALL) &&
+            !city_buildings_has_hippodrome()) {
```

Comment:
Can you add this for Julius as a PR too, please? I can do it too but you should get the credit.

Reply:
Will do. Should I keep this change in this PR as well?

Reply:
Yes, do keep it here please, it's one less thing to merge later on.